src/components/Spawn/spawnVolumeModal/getFormSchema.tsx

```diff
@@ -0,0 +1,177 @@
+import { css } from "@emotion/react";
+import { add } from "date-fns";
+import { GetFormSchema } from "components/SpruceForm/types";
+import { ExpirationRow } from "../ExpirationRow";
+import { getDefaultExpiration } from "../utils";
+
+interface Props {
+  maxSpawnableLimit: number;
+  availabilityZones: string[];
+  types: string[];
+  hosts: { id: string; displayName: string }[];
+  disableExpirationCheckbox: boolean;
+  noExpirationCheckboxTooltip: string;
+}
+
+export const getFormSchema = ({
+  maxSpawnableLimit,
+  availabilityZones,
+  types,
+  hosts,
+  disableExpirationCheckbox,
+  noExpirationCheckboxTooltip,
+}: Props): ReturnType<GetFormSchema> => ({
+  fields: {},
+  schema: {
+    type: "object" as "object",
+    properties: {
+      requiredVolumeInformation: {
+        type: "object" as "object",
+        title: "Required Volume Information",
+        properties: {
+          volumeSize: {
+            title: "Size (GB)",
```

Comment:
It seems slightly inconsistent to use `GB` in the title and `GiB` in the description. I must confess that I don't know which one is more correct 😬 but it seems like we've mostly gone with `GiB` w.r.t. volumes so maybe that should be our choice here.
```suggestion
            title: "Size (GiB)",
```